shared/js/background/chrome-events.es6.js

```diff
@@ -167,11 +167,15 @@ const redirect = require('./redirect.es6')
 const tabManager = require('./tab-manager.es6')
 const pixel = require('./pixel.es6')
 const https = require('./https.es6')
-const cookieConfig = require('./../background/storage/cookies.es6')
+const configStorage = require('./../background/storage/config.es6')
 
 const requestListenerTypes = utils.getUpdatedRequestListenerTypes()
 
 function blockTrackingCookies () {
+    if (configStorage.config.features) {
+        return configStorage.config.features.trackingCookies.state === 'enabled'
```

Comment:
Should we trust the config format to always have the expected structure under `features`? If the format changed, or a broken version of the list got shipped then this line could throw, effectively disabling cookie protection. Can we instead fail to a less broken state?